System/Console/Haskeline/Echo.hs

```diff
@@ -0,0 +1,147 @@
+{-# LANGUAGE CPP #-}
+
+{-|
+Module:      System.Console.Haskeline.Echo
```

Comment:
I prefer to avoid these headers in internal-only modules.  They also seem out of place (for example the LICENCE is actually BSD).